smalld/resources/ratelimit_buckets

```diff
@@ -0,0 +1,69 @@
+# Audit Log
```

Comment:
So as well as updating some \d to \w the \\ also had to be removed. Hmm.

It is actually cleaner without the \\. Will just have to write a customer parser for Java too (it's not a difficult format to parse tho)